.gitignore

```diff
@@ -1,5 +1,20 @@
+.idea
+vendor/
+
+# composer
 composer.lock
 composer.phar
-vendor/
+
+# osx
 .DS_Store
-.idea
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+
+# npm
+.npm/node_modules
+node_modules
+
+# generated files
+templates/theme/css
```

Comment:
hehe, well you want to generate a theme using scaffold, then do your adjustments there and port them to the scaffold template afterwards, else you will not be able to properly test and verify that things gets generated as you wish.